typing/printtyped.ml

```diff
@@ -69,9 +69,9 @@ let fmt_constant f x =
   | Const_string (s, strloc, Some delim) ->
       fprintf f "Const_string (%S,%a,Some %S)" s fmt_location strloc delim;
   | Const_float (s) -> fprintf f "Const_float %s" s;
-  | Const_int32 (i) -> fprintf f "Const_int32 %ld" i;
-  | Const_int64 (i) -> fprintf f "Const_int64 %Ld" i;
-  | Const_nativeint (i) -> fprintf f "Const_nativeint %nd" i;
+  | Const_int32 (_, s) -> fprintf f "Const_int32 %sl" s;
+  | Const_int64 (_, s) -> fprintf f "Const_int64 %sL" s;
+  | Const_nativeint (_, s) -> fprintf f "Const_nativeint %sn" s;
```

Comment:
Q: Would it be more useful to keep the previous behaviour (conversion to decimal) here?